### DISCUSSION TITLE
Cannot built libmesh with VTK

### DISCUSSION THREAD

**Discussion by omarlpzlpz** [libMesh/libmesh#3420] [Q&A]
Hello all,

I am trying to use libmesh with VTK, however I cannot build libmesh with
VTK, I attached the config.log file.
[config.log](https://github.com/libMesh/libmesh/files/9891252/config.log)

I am using this configuration command:

"../configure --prefix=/home/omar/libmesh/install\ --enable-distmesh\
--with-vtk-include=/opt/vtk/9.2.2/include/vtk-9.2\
--with-vtk-lib=/opt/vtk/9.2.2/lib/vtk-9.2\"

I am using Ubuntu 22.04.1 LTS and VTK 9.2.2.

I hope you can help me to find the error.

Thank you very much in advance.

Omar L.

**Comment by jwpeterson** [comment]
The output in your config.log looks strange to me:
```
configure:53435: checking /vtkConfigure.h usability
configure:53435: mpicxx -c   conftest.cpp >&5
conftest.cpp:202:10: fatal error: /vtkConfigure.h: No such file or directory
  202 | #include </vtkConfigure.h>
      |          ^~~~~~~~~~~~~~~~~
```

I don't understand why there is a leading slash in the filename that it is looking for. It is as though you passed
```
--with-vtk-include=
```
i.e. with nothing on the right-hand side of the equals sign. Configure is looking for the file
```
       AC_CHECK_HEADERS($VTK_INC/vtkConfigure.h, vtkincFound=yes)
```
and for some reason `$VTK_INC` is an empty string on your system.

Also, as I mentioned on the mailing list, you will most likely need the fix in #3346 in order to get VTK 9.2 working.
